dashboard/postgres_replicator/logger.go

```diff
@@ -0,0 +1,27 @@
+package transfer
+
+import (
+	"fmt"
+	"log"
+	"os"
+)
+
+// Logger proves per-table logging.
+type Logger struct {
+	*log.Logger
+}
+
+// NewLogger returns a new Logger.
+func NewLogger(tableName string) *Logger {
+	prefix := fmt.Sprintf("[%s] ", tableName)
+	prefixedLogger := log.New(os.Stderr, prefix, log.Ldate|log.Ltime|log.Lmsgprefix)
+	return &Logger{prefixedLogger}
+}
+
+// Errorf adds an "[ERROR]" tag to the log.
+func (tl *Logger) Errorf(format string, v ...interface{}) {
```

Comment:
I don't think there is a need to change the prefix here. You could instead change the format to 'Error: ' + format for errors.

Reply:
Fixed :+1: